src/components/Plugin/Plugin.tsx

```diff
@@ -149,8 +150,8 @@ export function PluginStore(props: PluginProps) {
         {subtitle}
         {subtitleIcon}
       </div>
-      <div className="line-clamp-2 mas-body2 max-h-11 text-f-primary">
-        {content}
```

Comment:
Why did you removed `line-clamp-2` ?

Reply:
it wasn't showing the correct num of chars

Reply:
Ok I see